src/ui/shared/offlineRound/view.tsx

```diff
@@ -22,7 +24,7 @@ function getChecksCount(ctrl: OfflineRoundInterface, color: Color) {
     return 0
 }
 
-export function renderAntagonist(ctrl: OfflineRoundInterface, content: Mithril.Children, material: Material, position: Position, isPortrait: boolean, otbFlip?: boolean, customPieceTheme?: string) {
+export function renderAntagonist(ctrl: OfflineRoundInterface, content: Mithril.Children, material: Material, position: Position, isPortrait: boolean, otbFlip?: boolean, customPieceTheme?: string, clock?: IChessClock | null) {
```

Comment:
Strange signature. You don't need the extra signification of `null` vs `undefined` here I suppose. So it should be `clock?: IChessClock`

Reply:
Agreed. At some point I was just trying to get things working and created this wacky signature. It's no longer necessary and has been adjusted to your suggested signature.